hack/samples/index.json

```diff
@@ -308,5 +308,10 @@
     "name": "Sample Nuxt.js Quote Generator",
     "description": "Quote Smith is a Nuxt.js app for creating and customizing inspirational quotes. Edit fonts, sizes, and gradients, then generate and download quotes as PNGs for sharing or personal use.",
     "gitUrl": "https://github.com/daytonaio/sample-nuxtjs-quote-generator"
+  },
+  {
+    "name": "Sudoku Game",
```

Comment:
Please set name to "Sample Next.js Sudoku Game"

Reply:
I have changed the name as requested. Thank you.